benchmarks/luafun/bench.lua

```diff
@@ -0,0 +1,178 @@
+local fun = require 'fun'
+local jitlog_capnp = require "jitlog_capnp"
+local bor = bit.bor
+local lshift = bit.lshift
+-- Enable table.new function
+require("table.new")
+
+local luafile = io.open("benchdata/jitlog.bin", "rb")

```

Comment:
Is there anything like the Python context manager in lua? No worries if not.

```
with open("file.txt") as fh:
    stuff = fh.read()
# file was closed when the context manager inner scope expired
```

Reply:
So there is nothing like this for Lua?

Reply:
Nope only exception catching stuff like pcall and xpcall.

Reply:
ok